frontend/src/components/user/ATSUserDetailsModal.vue

```diff
@@ -33,21 +32,16 @@ async function getATSClientInformation() {
     loading.value = true;
 
     const response = await atsService.searchATSUsers({
-      clientId: submission.atsClientNumber
+      clientId: atsClientNumber
     });
 
-    atsUsers.value = response.data.clients.map((client: any) => {
+    users.value = response.data.clients;
+
+    users.value.forEach((client: ATSClientResource) => {
       // Combine address lines and filter out empty lines
       const address = [client.address.addressLine1, client.address.addressLine2].filter((line) => line).join(', ');
-
-      return {
-        atsClientNumber: client.clientId,
-        firstName: client.firstName,
-        lastName: client.surName,
-        email: client.address.email,
-        phone: client.address.primaryPhone ?? client.address.secondaryPhone,
-        address: address
-      };
+      client.formattedAddress = address;
+      client.phone = client.address.primaryPhone ?? client.address.secondaryPhone;
```

Comment:
Is `client.phone` part of the original response? if not we should be referencing it through the proper keys instead of duplicating it.